classes/classes/PlayerAppearance.as

```diff
@@ -7,6 +7,23 @@ package classes
 	{
 		public function PlayerAppearance() {}
 		
+		private function feet_inch_and_metres (in_inches:Number):String {
+			if (flags[kFLAGS.USE_METRICS])  return "" + (Math.round(in_inches * 2.54) / 100).toFixed(2) + " metres";
+			else                            return "" + Math.floor(in_inches / 12) + " foot " + in_inches % 12 + " inch";
+		}
+
+		private function num_inches_and_centimetre (in_inches:Number):String {
+			var value = int(in_inches*precision * (flags[kFLAGS.USE_METRICS] ? 2.54 : 1)) / precision;
```

Comment:
this method is lacking the precision param your using below
And personally I'd use `, precision:int = 1)` since precision usually refers to the number of digits after the decimal seperator.
and then use Math.pow(10, precision)